docs/source/pillars/computing_technology.md

```diff
@@ -33,8 +33,9 @@ _The interfaces used for interacting with the TRE management system and the TRE
     In practice a web browser is the most common way to achieve this.
   - Recommended
 * - A TRE workspace should provide an environment familiar to the users of the TRE.
-  - This may be in the form of a virtual Windows or Linux desktops, web applications, or a terminal.
-    The use of custom developed TRE-specific software should be avoided when widely used open-source alternatives already exist.
+  - This may take the form of a virtual Windows or Linux desktops, non-desktop interfaces such as JupyterLab and other web applications, or a terminal.
+    Alernatively, this could be a GUI with intuitive design features relevant to the TRE users (e.g. scientific domain or dataset type specific features for non-programmer scientists).
+    Custom-developed TRE-specific software should however be avoided when widely used open-source alternatives already exist.
```

Comment:
I have rephrased this whole table row, although I'm still not sure about the part in brackets, could I get away with removing it or does that make it to vague?

Reply:
I'm not sure this makes sense now because it reads like,

1. The environment should be familiar
2. Could be something familiar (desktop, Jupyter, terminal, ...)
3. Could be something bespoke <- but this is an example of something unfamiliar

Maybe we are trying to capture the benefits of 'familiarity' and 'suitability/efficiency' in one statement. Perhaps it would be better as two.

Reply:
I don't see what the GUI example adds. Saying GUI and 'intuitive design features' feels like a specific suggestion but I'm not sure what it is referring to. That isn't how I would describe OpenSAFELY.

Reply:
Probably better to say 'bespoke' than 'Custom-developed'.

Reply:
> I don't see what the GUI example adds. Saying GUI and 'intuitive design features' feels like a specific suggestion but I'm not sure what it is referring to.

The idea is that the interface should feel familiar because it is either a) a desktop or b) a GUI that the researchers will understand - do you think there is a better way to express this?